node/db/src/rocks.rs

```diff
@@ -158,3 +159,35 @@ impl Blockstore for RocksDb {
         self.bulk_write(&values).map_err(|e| e.into())
     }
 }
+
+impl BitswapStore for RocksDb {
+    /// `fvm_ipld_encoding::DAG_CBOR(0x71)` is covered by [`libipld::DefaultParams`]
+    /// under feature `dag-cbor`
+    type Params = libipld::DefaultParams;
+
+    fn contains(&mut self, cid: &Cid) -> anyhow::Result<bool> {
+        Ok(self.exists(cid.to_bytes())?)
+    }
+
+    fn get(&mut self, cid: &Cid) -> anyhow::Result<Option<Vec<u8>>> {
+        Blockstore::get(self, cid)
+    }
+
+    fn insert(&mut self, block: &libipld::Block<Self::Params>) -> anyhow::Result<()> {
+        self.put_keyed(block.cid(), block.data())
+    }
+
+    fn missing_blocks(&mut self, cid: &Cid) -> anyhow::Result<Vec<Cid>> {
```

Comment:
Can we extract this method? Seems it will be exactly the same for all structs implementing `Store`.

Reply:
Good suggestion, fixed